schema/all_test.go

```diff
@@ -42,16 +46,26 @@ func (tc fieldValidatorTestCase) Run(t *testing.T) {
 
 		if cmp, ok := tc.Validator.(schema.Compiler); ok {
 			err := cmp.Compile(tc.ReferenceChecker)
-			assert.NoError(t, err, "Validator.Compile(%v)", tc.ReferenceChecker)
+			if err != nil {
+				t.Errorf("Validator.Compile(%v): unexpected error: %v", tc.ReferenceChecker, err)
+			}
 		}
 
 		v, err := tc.Validator.Validate(tc.Input)
 		if tc.Error == "" {
-			assert.NoError(t, err, "Validator.Validate(%v)", tc.Input)
-			assert.Equal(t, tc.Expect, v, "Validator.Validate(%v)", tc.Input)
+			if err != nil {
+				t.Errorf("Validator.Validate(%v): unexpected error: %v", tc.ReferenceChecker, err)
+			}
+			if !reflect.DeepEqual(v, tc.Expect) {
+				t.Errorf("Validator.Validate(%v): expected: %v, got: %v", tc.Input, tc.Expect, v)
+			}
 		} else {
-			assert.EqualError(t, err, tc.Error, "Validator.Validate(%v)", tc.Input)
-			assert.Nil(t, v, "Validator.Validate(%v)", tc.Input)
+			if err.Error() != tc.Error {
+				t.Errorf("Validator.Validate(%v): return error: %v, wanted: %v", tc.ReferenceChecker, tc.Error, err)
```

Comment:
swapped order fo got / expected compared to other Errorfs

Reply:
actually the order is the same for tje args, but the text in the format string is turned